pkg/asset/manifests/aws/cluster.go

```diff
@@ -31,6 +31,14 @@ func GenerateClusterAssets(ic *installconfig.InstallConfig, clusterID *installco
 		return nil, fmt.Errorf("failed to get user tags: %w", err)
 	}
 
+	sshRuleCidr := []string{"0.0.0.0/0"}
+	if ic.Config.Publish == types.InternalPublishingStrategy {
+		sshRuleCidr = []string{}
+		for _, machineNetwork := range ic.Config.Networking.MachineNetwork {
+			sshRuleCidr = append(sshRuleCidr, machineNetwork.CIDR.String())
+		}
```

Comment:
This could be useful for other platforms too. What about adding it to `pkg/asset/manifests/capiutils/helpers.go`. We already have `CIDRFromInstallConfig` in there.

Reply:
Hm I think I will just use CIDRFromInstallConfig. We don't support multiple machine networks ATM